common/src/main/java/io/netty/util/ResourceLeak.java

```diff
@@ -16,6 +16,10 @@
 
 package io.netty.util;
 
+/**
+ * Deprecated please use {@link ResourceLeakTracker}.
```

Comment:
Add a warning as to this API (specifically close) may result in false positives.